event.c

```diff
@@ -2133,7 +2133,9 @@ event_base_once(struct event_base *base, evutil_socket_t fd, short events,
 			mm_free(eonce);
 			return (res);
 		} else {
-			LIST_INSERT_HEAD(&base->once_events, eonce, next_once);
+			if (base) {
```

Comment:
So simply ignore the invalid calls? This is no good.

Other places have the following:

```c
    if (!base)
        base = current_base;
```

But I would prefer simply to fail, return `-1` w/o base